packages/core/src/lib/select/__tests__/searchable-select.spec.ts

```diff
@@ -192,6 +223,27 @@ describe('SearchableSelect', () => {
     expect(search).not.toHaveAttribute('aria-activedescendant');
   });
 
+  it('has options with labels descriptions and icons', () => {
+    renderSubject({
+      options: detailedOptions,
+    });
+
+    const { search, options } = getResults();
+
+    expect(options).toHaveLength(2);
+    const firstOption = screen.getByRole('option', { name: /gale/iu });
+    const secondOption = screen.getByRole('option', { name: /karlach/iu });
+    expect(firstOption).toHaveAttribute('aria-selected', 'false');
+    expect(secondOption).toHaveAttribute('aria-selected', 'false');
+    expect(search).not.toHaveAttribute('aria-activedescendant');
+    // icons are rendered
+    expect(screen.getByTestId('icon-viam-process')).toBeInTheDocument();
+    expect(screen.getByTestId('icon-language-cpp')).toBeInTheDocument();
+    // descriptions are rendered
+    expect(screen.getByText(/wizard/iu)).toBeInTheDocument();
+    expect(screen.getByText(/barbarian/iu)).toBeInTheDocument();
```

Comment:
I think we can use these tests to guide better accessibility for the UI:

```suggestion
    expect(firstOption).toHaveAccessibleDescription(/wizard/iu);
    expect(secondOption).toHaveAccessibleDescription(/barbarian/iu);
```

Reply:
23d95db